src/com_tjqueue/media/helpers/tjqueuecontext.php

```diff
@@ -0,0 +1,96 @@
+<?php
```

Comment:
Move this as a class in libraries folder in the component admin folder

Reply:
As per discussion after install tjqueue library will be installed in:  administrator/components/com_tjqueue/libraries/

Reply:
This is done